infrastructure/helm-chart/charts/core/charts/components/charts/api/charts/api-admin/templates/deployment.yaml

```diff
@@ -61,6 +61,16 @@ spec:
             configMapKeyRef:
               name: beanstalk-config
               key: BEANSTALK_PORT
+        - name: CORE_ID
```

Comment:
I'd propose to reference this like the security config map:
```yaml
envFrom:
  - configMapRef:
     name: security
```
to avoid redundantly spelling out variables that every app may depend on.